### ISSUE TITLE
Psensor status icons

### ISSUE THREAD

**Issue by dirtydancing** [numixproject/numix-core#864]
Hi, in Ubuntu, the Psensor status icons in Numix-Circle are in need of correction (Ubuntu 14.04 LTS with Psensor 1.0.2).

Psensor has an application icon in Numix-Circle, psensor.svg. This icon is showing correctly in the Ubuntu Dash and Launcher.
![screenshot0](https://cloud.githubusercontent.com/assets/7164227/3236026/12d6f846-f0d9-11e3-8c05-70cf6d858f6d.png)

At the same time, Psensor comes with two status icons, psensor_normal.svg and psensor_hot.svg.
![screenshot1](https://cloud.githubusercontent.com/assets/7164227/3236032/1975b6ce-f0d9-11e3-8ee7-3efb7fc76759.png)
![screenshot2](https://cloud.githubusercontent.com/assets/7164227/3236035/1e8f0d86-f0d9-11e3-9012-556ea415541c.png)

At present, there is one Psensor status icon in Numix-Circle, called psensor.svg.
![screenshot3](https://cloud.githubusercontent.com/assets/7164227/3236046/5d3216dc-f0d9-11e3-876e-8b249ad9a861.png)

This status icon does not show in the Ubuntu menu bar, but rather the original status icon which is in hicolor.
![screenshot4](https://cloud.githubusercontent.com/assets/7164227/3236053/79ec2970-f0d9-11e3-8d85-ac6601accb4a.png)

After adding this icon as psensor_normal.svg to Numix-Circle, the correct Numix-Circle status icon is displayed.
![screenshot5](https://cloud.githubusercontent.com/assets/7164227/3236066/a04c51b2-f0d9-11e3-8fa7-30b7675a29bd.png)

In conclusion, in Numix-Circle, the status icon psensor.svg needs to be renamed in psensor_normal.svg, and the second status icon psensor_hot.svg needs to be created.
